scripts/tag-release.mjs

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console */
+
+import {execSync} from 'node:child_process';
+import {readFile} from 'node:fs/promises';
+import {URL} from 'node:url';
+
+import program from 'commander';
+
+async function main(tag) {
+  let publishSummary = JSON.parse(
+    await readFile(
+      new URL('../lerna-publish-summary.json', import.meta.url).pathname,
```

Comment:
I haven't seen this kind of pattern of using an URL in a `readFile`, what does it do over a `readFile(path.resolve(__dirname, "../lerna-publish-summary.json"))`?

Reply:
Neither, only just used it for the first time. It's the equivalent to your example in esm land, and will be an absolute path due to the `pathname`

Example: `/parcel/lerna-publish-summary.json`